core/dogu_v2.go

```diff
@@ -601,6 +602,16 @@ type Dogu struct {
 	//  VERSION="1.23.2-1"
 	//
 	Version string
+	// PublishedAt is the date and time when the dogu was created.
+  //
+  // This field does not need to be filled in by the developer. The dogu.cloudogu.com service automatically replaces
+  // the content with the current time when the new dogu is created.
+  //
+  // Examples:
+  //   - 2024-10-16T07:49:34.738Z
+  //   - 2019-05-03T13:31:48.612Z
+  //
+	PublishedAt time.Time
```

Comment:
fix indentation

Reply:
Indentation should be fixed in the latest commit. There was a moment where prettier stopped working and proper formatting didn't happen on save.